cyberark-secrets-provider-for-kubernetes/templates/role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if eq .Values.rbac.serviceAccount.create true -}}
```

Comment:
I think this should work without specifying `true` because it's a boolean condition